### PR TITLE
Week 1 sarah

### PR DESCRIPTION
**Pull request by sjproctor**
- Interview questions as a team, so no "your answer"
- Didn't do the number 6 interview, as it is an individual question
- Code conceptual challenges
- Code function challenges with many options and pseudocoding

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -14,6 +14,25 @@ var temp1 = 35
...
+
+const boilingPoint = (temp) => {
+  if(temp === 212) {
+    return "212 is at boiling point"
+  } else if(temp > 212) {
+    return `${temp} is above boiling point`
+  } else if(temp < 212) {
+    return `${temp} is below boiling point`
+  }
+}
```
Comment (WalkerTexas-coder) [inline code-challenges.js:30]: Looks great!

File: code-challenges.js [hunk 2]
```diff
@@ -22,6 +41,15 @@ var temp3 = 212
 var myNumbers1 = [3, 7, 0, 36, -9]
 var myNumbers2 = [8, -7, 22, 9, 13]
 
+// create a function called sortedArray
+// parameters - nums1, nums2
+// declared a new vaiable to store the .concat combined arrays
+// return the array with .sort() to get the numbers from least to greatest
+
+const sortedArray = (nums1, nums2) => {
+  return nums1.concat(nums2).sort((firstNumber, secondNumber) => firstNumber - secondNumber)
```
Comment (WalkerTexas-coder) [inline code-challenges.js:50]: I like the use of specific values!

File: code-challenges.js [hunk 3]
```diff
@@ -30,7 +58,28 @@ var myNumbers2 = [8, -7, 22, 9, 13]
...
+// const reverseString = (string) => {
+//   let newArray = string.split("")
+//   console.log(newArray)
+//   let reversedArray = newArray.reverse()
+//   console.log(reversedArray)
+//   let joinedArray = reversedArray.join("")
+//   console.log(joinedArray)
+//   return joinedArray
+// }
+
```
Comment (WalkerTexas-coder) [inline code-challenges.js:77]: I see why you have both! Good work!

File: code-challenges.js [hunk 4]
```diff
@@ -52,3 +127,33 @@ var numberExampleRefactor2 = "hello"
...
+// const subtractor = (num1, num2) => {
+//   if(typeof num1 !== "number" || typeof num2 !== "number") {
+//     return "Your input is not a number"
+//   } else if(num1 >= num2) {
+//     return num1 - num2
+//   } else {
+//     return num2 - num1
+//   }
+// }
+
```
Comment (sjproctor) [inline code-challenges.js:146]: Very curious about the typeof operator here

File: code-challenges.js [hunk 5]
```diff
@@ -52,3 +127,33 @@ var numberExampleRefactor2 = "hello"
...
 var numberExampleRefactor3 = 27
 var numberExampleRefactor4 = 24
+
+// Copied the code from the function that just handles numbers
+// Refactor: Check if the arguments are numbers
+// typeof operator
+// Return a statement if the arguments are not numbers
+
+
+// const subtractor = (num1, num2) => {
```
Comment (WalkerTexas-coder) [inline code-challenges.js:137]: Way Cool!

**Review by WalkerTexas-coder — APPROVED** [review]
Sarah and Class great work! Very creative options for the code! I really appreciate the pseudo code! your PR is approved!

**Pull request merged**